PowerShell Scanners/RDP Last Logoff/RDP_Last_Logoff.ps1

```diff
@@ -0,0 +1,39 @@
+$LogName = 'Microsoft-Windows-TerminalServices-LocalSessionManager/Operational'
+$Results = @()
+$Events = Get-WinEvent -LogName $LogName
+$UserArray = New-Object System.Collections.ArrayList
+
+#Parse WinEvent log for info
+foreach ($Event in $Events) {
```

Comment:
The variable `$Event` is a special variable in PowerShell, would you change this to something like `$log in $logs` or `$item in $items`? Anything you think fits.

https://docs.microsoft.com/en-us/powershell/module/microsoft.powershell.core/about/about_automatic_variables?view=powershell-7.1#event